Doc/library/sys.rst

```diff
@@ -1392,7 +1396,12 @@ always available.
    ``'return'``, ``'c_call'``, ``'c_return'``, or ``'c_exception'``. *arg* depends
    on the event type.
 
-   .. audit-event:: sys.setprofile "" sys.setprofile
+.. note::
+   The same tracing mechanism is used for :func:`!setprofile` as :func:`settrace`.
+   To trace calls with :func:`!setprofile` inside a tracing function (e.g. in a
+   debugger breakpoint) see :func:`call_tracing`.
+
+.. audit-event:: sys.setprofile "" sys.setprofile
```

Comment:
Hey, sorry if I was unclear before—the `note` should go one block above, and the `audit-event` should go where the `note used to be (i.e. below the list of *event* values. This will avoid interjecting these two between the introduction of the profile function args and the description of their values, which is particularly confusing to readers because the audit "events" are entirely unrelated to the *event* argument here.

Also, the indents got off as a result of this change.

As I should have before, here's a patch you can apply with

```shell
git apply 0001-Move-call_tracing-note-audit-event-in-setprofile-to-.patch
```
to fix all this directly:

[0001-Move-call_tracing-note-audit-event-in-setprofile-to-.patch](https://github.com/python/cpython/files/11010680/0001-Move-call_tracing-note-audit-event-in-setprofile-to-.patch)